next-env.d.ts

```diff
@@ -1,2 +1,4 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
+
+/* declare module "*.css"; */
```

Comment:
Why have you added this?

Reply:
I was trying to fix the  import bug.  It didn't work, so I'd rather delete this line actually.